raiden_contracts/data/source/raiden/TokenNetwork.sol

```diff
@@ -1161,6 +1157,60 @@ contract TokenNetwork is Utils {
         return keccak256(abi.encodePacked(channel_identifier, sender, receiver));
     }
 
+    function setTotalDepositHelper(
```

Comment:
I don't like the name too much, as the "helper" does all of the work, not just some help. Maybe `setTotalDepositFor`>